### PR TITLE
I have completed the assignment

### PR DESCRIPTION
**Pull request by krischal111**
All the code worked as expected upon testing.

### CODE REVIEW

File: Day0/krischal111/Assignment1/solution.c [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    struct coordinate start, end;
+};
+
+/* Functions used*/
+int squareof(int number)
+{
+    return number*number;
+}
+
+char  writepixel(void * image_data, struct coordinate size, struct coordinate coord, struct pixel pxl)
```
Comment (bhattaraiashish) [inline Day0/krischal111/Assignment1/solution.c]: Why do you return the same same from the function always?

File: Day0/krischal111/assignment2/solution.c [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    {
+        c = fgetc(myfile);
+    } while (c == ' ') ;
+    
+    return c;
+}
+
+int main()
+{
+    FILE * myfile = fopen(expressionfile, "r");
```
Comment (bhattaraiashish) [inline Day0/krischal111/assignment2/solution.c]: Recommended to Read/Open file as binary i.e. "rb" and not "r" always, same for writing. More on that here: https://stackoverflow.com/questions/229924/difference-between-files-written-in-binary-and-text-mode

File: Day0/krischal111/assignment2/solution.c [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	    
+	    if(c == EOF)
+	        break;
+	
+	    float a = getnum(myfile);
+	    while(c != '\n' && c != EOF)
+	    {
+	        int b = getnum(myfile);
+	        
+	        switch(getoperator(myfile))
```
Comment (bhattaraiashish) [inline Day0/krischal111/assignment2/solution.c]: This program doesn't work for all postfix expressions, such as "5 6 7 + *"

**Comment by bhattaraiashish** [comment]
Please remove examples and generated file before we can accept the pull request

File: Day0/krischal111/assignment2/solution.c (hunk 2 above)
Comment (krischal111) [reply Day0/krischal111/assignment2/solution.c]: Okay. Didn't know that reading text in binary mode was okay, but not vice versa. I've corrected that now.

File: Day0/krischal111/assignment2/solution.c (hunk 3 above)
Comment (krischal111) [reply Day0/krischal111/assignment2/solution.c]: Resolved (Had to learn, make and use custom stack, but works now + also shows calculation steps)

File: Day0/krischal111/Assignment1/solution.c (hunk 1 above)
Comment (krischal111) [reply Day0/krischal111/Assignment1/solution.c]: Made the function void type.

**Pull request merged**